### ISSUE TITLE
영속성 관리 3.3

### ISSUE THREAD

**Issue by Brylimo** [Brylimo/TIL#148]
## 영속성 관리 3.3
_엔티티의 생명주기_

⭐ 엔티티에는 4가지 상태가 존재한다.
- 비영속(new/transient): 영속성 컨텍스트와 전혀 관계가 없는 상태
    - 지금은 순수한 객체 상태이며 아직 저장하지 않았다.
    - 따라서 영속성 컨텍스트나 데이터베이스와는 전혀 관련이 없다.
- 영속(managed): 영속성 컨텍스트에 저장된 상태
    - 엔티티 매니저를 통해서 엔티티를 영속성 컨텍스트에 저장했다.
    - 이렇게 영속성 컨텍스트가 관리하는 엔티티를 영속 상태라 한다.
    - 영속 상태라는 것은 영속성 컨텍스트에 의해 관리된다는 뜻이다.
    - `em.find()`나 JPQL을 사용해서 조회한 엔티티도 영속성 컨텍스트가 관리하는 영속 상태다.
- 준영속(detached): 영속성 컨텍스트에 저장되었다가 분리된 상태
   - 영속성 컨텍스트가 관리하던 영속 상태의 엔티티를 영속성 컨텍스트가 관리하지 않으면 준영속 상태가 된다.
   - 특정 엔티티를 준영속 상태로 만들려면 `em.detach()`를 호출하면 된다.
- 삭제(removed): 삭제된 상태
   - 엔티티를 영속성 컨텍스트와 DB에서 삭제한다. (em.remove())
![image](https://user-images.githubusercontent.com/48869794/226919048-5bac6c08-5835-4a33-b68c-177873f6096a.png)
